debian/kafka-connect/Dockerfile

```diff
@@ -39,3 +39,5 @@ RUN echo "===> Installing JDBC, Elasticsearch and Hadoop connectors ..." \
 
 RUN echo "===> Installing GCS Sink Connector ..."
 RUN confluent-hub install confluentinc/kafka-connect-gcs:latest --no-prompt
+
+ENV CONNECT_PLUGIN_PATH=/usr/share/java/,/usr/share/confluent-hub-components/
```

Comment:
nit: Please add a newline at the end of the file (here and for the one within `server-connect` too)

Reply:
Done